packages/suite/src/views/suite/settings/index.tsx

```diff
@@ -1,14 +1,307 @@
-import React from 'react';
+/* eslint-disable @typescript-eslint/camelcase */
+import React, { useEffect, useState } from 'react';
 import styled from 'styled-components';
+import { Input, Button, P, Icon, Select } from '@trezor/components';
+import { AppState } from '@suite-types/index';
+import { resolveStaticPath } from '@suite-utils/nextjs';
+import elementToHomescreen from '@suite/utils/suite/elementToHomescreen';
+
+const IMAGE_NAMES_T1 = [
```

Comment:
move to constants

Reply:
As you wish. I am not a furious advocade of code splitting in cases when variables might be used only in place where they are defined and nowhere else, which is the case. On the other hand, settings/index.tsx is going to grow with time, so moving these variables to constants might be desirable.